.github/workflows/BuildAndRun.yaml

```
@@ -75,26 +75,42 @@ jobs:
           colcon mixin update default
         shell: bash
 
-      - name: Install sonar-scanner and build-wrapper
-        uses: sonarsource/sonarcloud-github-c-cpp@v3
+      - name: Install Build Wrapper
+        if: ${{ matrix.resource != ubuntu-22.04 }} # @TODO Support unit/integration test with ARM CPU.
```

```suggestion
        if: ${{ matrix.runs_on != ubuntu-22.04 }} # @TODO Support unit/integration test with ARM CPU.
```

The all if statements may be wrong.
Please make sure that the action has been passed.

https://github.com/tier4/scenario_simulator_v2/actions/runs/14393607085?pr=1512